docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+# Blog tutorial
+
+## 1. Overview
```

Comment:
Albeit I understand the point of this section, it's a little off-putting to me because it talks so much about what the reader is going to learn. We can leave it like that and test it, but if we can somehow subsum this in a few sentences it will likely feel much lighter.

It's kind of similar to what you criticized in the plugins section a while ago – too much introductory prose that is not relevant to what the user wants to achieve at that point. However, I think that the categories (prerequisites, time required, etc.) are a good idea, but maybe we can keep it more concise.

Reply:
Additionally, I'm not sure we should number sections. It looks very weird and if we, at some point, add a section in-between, all subsequent fragments will be broken. Thus removing the numbers allows us to keep anchors stable more easily.

Reply:
> talks so much about what the reader is going to learn

This can certainly be tightened up language-wise. The "outcomes" can probably go when the tutorial is split up as you suggested and perhaps the prerequisites are generic, so could go onto the main tutorial page? The time requirement can probably be just given as "ca. XX minutes"?

> I'm not sure we should number sections

Agreed. I put the numbers in because I thought that it would be easier to find a section that is cross-referenced. I imagined that we would automate the numbering, did not want to maintain that, either!

Reply:
> This can certainly be tightened up language-wise. The "outcomes" can probably go when the tutorial is split up as you suggested and perhaps the prerequisites are generic, so could go onto the main tutorial page? The time requirement can probably be just given as "ca. XX minutes"?

Yes, a main tutorial page is a good idea where we explain how we write tutorials and what to expect. Let's postpone that to after the second tutorial, so we know at least some commonalities. However, I also agree that a short motivating 3-liner is always good to set the stage, which is why our documentation features those 3-liners basically on almost every page. To sum up: a great idea, but IMHO we should check if we can find a better cut.

> Agreed. I put the numbers in because I thought that it would be easier to find a section that is cross-referenced. I imagined that we would automate the numbering, did not want to maintain that, either!

We can certainly automate the numbering presentation-wise with [CSS counters](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_counter_styles/Using_CSS_counters).